sources/packages/forms/src/form-definitions/designationagreementdetails.json

```diff
@@ -3012,6 +3018,32 @@
               "hideLabel": true,
               "tags": []
             },
+            {
+              "key": "officerDuplicateWarningHtml",
+              "label": "HTML",
+              "input": false,
+              "tag": "p",
+              "attrs": [
+                {
+                  "value": "",
+                  "attr": ""
+                }
+              ],
+              "className": "alert alert-warning fa fa-exclamation-triangle w-100",
+              "content": "You have the same official listed as both an Eligibility Officer and an Enrolment Officer. StudentAid BC policy is that these roles must be effectively segregated such that no employee is appointed to perform both duties. An exemption may be granted upon written request and subject to Ministry approval, if a designated institution had fewer than 10 student borrowers who received a combined total of under $50,000 in student financial assistance in the previous program year.\nPlease email DESIGNAT@gov.bc.ca if you want to request an exemption.",
```

Comment:
I am not sure if it is a blocker but in the business AC the `DESIGNAT@gov.bc.ca` is rendered as a hyperlink.

Reply:
updated